### DISCUSSION TITLE
what is your prefered pkg choice for this project?

### DISCUSSION THREAD

**Discussion by osevan** [roddhjav/apparmor.d#227] [General]
For arch users apparmor-git?

Or apparmor pkg?

Because i want test both and make issue to be sure if nothing of packages break something. 

Thanks and 

best regards

**Comment by roddhjav** [comment]
For arch based distribution both are fine, there is no difference for this project between [`apparmor`](https://archlinux.org/packages/extra/x86_64/apparmor/) and [`apparmor-git`](https://aur.archlinux.org/packages/apparmor-git)
